src/@utils/cors.ts

```diff
@@ -0,0 +1,21 @@
+const allowCors = (fn) => async (req, res) => {
+  res.setHeader('Access-Control-Allow-Credentials', true)
+  res.setHeader('origin', 'https://nextjs-graphql-server-client.vercel.app')
```

Comment:
Is it needed to do this from the app, instead of from the env hosting the app?

The vercel is awfully specific and should either be configurable (and thus also allow for multiple values), or it should be configured on the hosting environment

Reply:
Removed!